lib/screenplay_web/auth_manager/auth_manager.ex

```diff
@@ -24,21 +26,26 @@ defmodule ScreenplayWeb.AuthManager do
 
   def resource_from_claims(_), do: {:error, :invalid_claims}
 
-  @spec claims_access_level(Guardian.Token.claims()) :: access_level()
+  @spec claims_access_level(Guardian.Token.claims()) :: list(access_level())
```

Comment:
Are there no tests for this module? I guess not, otherwise they would have had to change here. Seems like something we should have given the importance of getting access control right. (I think Guardian has some helper functions for testing with claims)

Reply:
There are tests for the ensure plug (realizing now that the new one does not have tests) but not this module specifically. Those tests use tags to add user and roles and then asserts on the return of the plug itself. Do you think that is enough?